betScanner.py

```diff
@@ -0,0 +1,51 @@
+import time
+from datetime import datetime
+
+import logging
+
+from selenium import webdriver
+
+
+def main():
+    driver = webdriver.Chrome()
+    driver.get("https://www.marathonbet.ru/su/")
+    btn_tennis = driver.find_element_by_class_name("icon-sport-tennis")
+    btn_tennis.click()
+    start_time = time.time()
+    file_log = logging.FileHandler('Log.log')
+    console_out = logging.StreamHandler()
+    logging.basicConfig(handlers=(file_log, console_out), format='[%(asctime)s | %(levelname)s]: %(message)s',
+                        datefmt='%m.%d.%Y %H:%M:%S', level=logging.INFO)
+    mathes = []
+    table_mass=scrollToTheEnd(driver)
+    for i in table_mass:
```

Comment:
for loop is okay but when you need to convert one array into another consider using map function. lambda is more modern approach.